### PR TITLE
Fix getFirstCommitAfterForkingFromMain-test

### PR DESCRIPTION
**Pull request by lunaleaps**
I think a rename from master -> main forgot this

### CODE REVIEW

**Comment by lunaleaps** [comment]
Tried running `yarn test` but failing with this -- is there anything I'm missing? 
```
 FAIL  scripts/__tests__/changelog-generator.test.ts
  ● Test suite failed to run

    Cannot find module '/Users/luna/releases/node_modules/@babel/compat-data/data/corejs3-shipped-proposals'

      at createEsmNotFoundErr (node:internal/modules/cjs/loader:960:15)
      at finalizeEsmResolution (node:internal/modules/cjs/loader:953:15)
      at resolveExports (node:internal/modules/cjs/loader:482:14)
      at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:94:18)
      at Object.<anonymous> (node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js:10:55)
      at Module._compile (node:internal/modules/cjs/loader:1101:14)
```

**Comment by tido64** [comment]
Oh, interesting… If you open up `node_modules/@babel/compat-data/package.json` and modify line 15:

```diff
-    "./corejs3-shipped-proposals": "./data/corejs3-shipped-proposals",
+    "./corejs3-shipped-proposals": "./data/corejs3-shipped-proposals.json",
```

The tests should run. We probably just need to bump the latest version to fix this.

File: scripts/__tests__/changelog-generator.test.ts [hunk 1]
```diff
@@ -70,13 +70,11 @@ describe(getOriginalCommit, () => {
   });
 });
 
-describe(getFirstCommitAfterForkingFromMaster, () => {
+describe(getFirstCommitAfterForkingFromMain, () => {
   it("returns the SHA of the first commit where its first parent is on the master branch", () => {
```
Comment (tido64) [inline scripts/__tests__/changelog-generator.test.ts:74]: Can you also fixup all refs to master while you're here? 😄 
```suggestion
  it("returns the SHA of the first commit where its first parent is on the main branch", () => {
```

**Review by tido64 — APPROVED** [review]
Nice! Tests aren't run on CI so we missed this entirely.

**Pull request merged**